### DISCUSSION TITLE
New Documentation?

### DISCUSSION THREAD

**Discussion by Bloeckchengrafik** [Minestom/Minestom#1969] [General]
Hello everyone,
there were some discussions in the discord about rebuilding the documentation in something other than GitBook since it's not easy to edit people without access to the web UI. 

In my opinion, the new documentation website should:
- be easy to work on
- build to static pages

Tools like Docusaurus, mkdocs, Writerside or other static page generators should work for this.

But what should we use? If you got any preferences or ideas, feel free to post them here!

**Comment by Tricked-dev** [comment]
https://starlight.astro.build/ works pretty well and it uses [Astro](https://astro.build/) so it can easily be used to make the main website too

**Reply by Bloeckchengrafik** [reply]
That seems like a good idea! The current website is written with Gatsby but if we want to integrate them, we could follow this [migration guide](https://docs.astro.build/en/guides/migrate-to-astro/from-gatsby/).
